### PR TITLE
preview provider for emf

### PR DESCRIPTION
**Pull request by kesselb**
### ☑️ Resolves

- Move web server configuration update up because it's important
- Document https://github.com/nextcloud/server/pull/41395

### 🖼️ Screenshots

image

### CODE REVIEW

File: admin_manual/release_notes/upgrade_to_28.rst [hunk 1]
```diff
@@ -9,6 +9,11 @@ System requirements
 
 To be documented
 
+Web server configuration
+------------------------
+
+* The recommended :ref:`nginx configuration<nginx-config>` changed as Nextcloud Talk now serves audio files with ``.ogg`` / ``.flac`` extension, make sure to add these extensions to the list of static files.
```
Comment (ChristophWurst) [inline admin_manual/release_notes/upgrade_to_28.rst:15]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged :sparkles:

File: admin_manual/release_notes/upgrade_to_28.rst [hunk 2]
```diff
@@ -18,7 +23,20 @@ You can still ask the monitoring endpoint to show new app updates by using the U
...
 https://github.com/nextcloud/serverinfo#api
 
-Web server configuration
-------------------------
+Previews for Office files using LibreOffice
+-------------------------------------------
+
+Nextcloud can generate previews for Office files using LibreOffice.
+
+Since Nextcloud 28, you can also create previews for EMF files.
```
Comment (ChristophWurst) [inline admin_manual/release_notes/upgrade_to_28.rst:31]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded :sparkles:

**Review by ChristophWurst — APPROVED** [review]

**Comment by kesselb** [comment]
Add a note about the richdocuments app preview provider for emf based on collabora.

**Comment by kesselb** [comment]
Possible follow-up: 

Copy some text over to https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/previews_configuration.html#parameters.

**Pull request merged**